integrations/dav-server/Cargo.toml

```diff
@@ -35,7 +35,7 @@ dav-server = { version = "0.5.8" }
 dirs = "5.0.0"
 futures = "0.3"
 futures-util = { version = "0.3.16" }
-opendal.workspace = true
+opendal = { path = "../../core"}
```

Comment:
Ditto https://github.com/apache/opendal/pull/4134/files#r1477020121.

Reply:
The same with https://github.com/apache/opendal/pull/4134#discussion_r1477048529